rasa/core/policies/rule_policy.py

```diff
@@ -71,6 +94,8 @@ def __init__(
         core_fallback_threshold: float = 0.3,
         core_fallback_action_name: Text = ACTION_DEFAULT_FALLBACK_NAME,
         enable_fallback_prediction: bool = True,
+        restrict_rules: bool = True,
+        check_rules_with_stories: bool = True,
```

Comment:
I'd prefer this one as it explains what is checked and doesn't expose to much of the "how"
```suggestion
        check_for_contradictions: bool = True,
```